pkg/virt-handler/vm_test.go

```diff
@@ -3301,6 +3304,245 @@ var _ = Describe("VirtualMachineInstance", func() {
 		})
 	})
 
+	Context("setMigrationProgressStatus", func() {
+		newDomainMigrationKubevirtMetadata := func(miguid types.UID, end *metav1.Time, completed, failed bool, mode v1.MigrationMode) *api.Domain {
```

Comment:
Does it need to be a lambda?

Reply:
+1

Reply:
@0xFelix how would you prefer it?

Reply:
Can you move it out of the Context to the bottom of the file and make it `func newDomainMigrationKubevirtMetadata(miguid types.UID, end *metav1.Time, completed, failed bool, mode v1.MigrationMode) *api.Domain {...}`?

Reply:
it is related to this context and not to the general suite, do we have guidelines for this?

Reply:
Coding conventions say this:

```
Use closures with caution, be aware of the risks and use them only when it makes sense.

    Acceptable use cases are:
        Defined and used inline, not though variable assignment.
        Used as actual closures, binding to external variables.
        Grouping a set of instructions to provide common scope functionality (e.g. defer).
    Exceptions may exist, but they need to be well reasoned for, expressing what alternatives have been considered.
```

Now it is up for debate if grouping to context makes sense or not. :)